lib/core/include/irods/msParam.h

```diff
@@ -94,8 +94,9 @@ typedef struct ValidKeyWd {
 
 #define RESC_NAME_FLAG          0x1
 #define DEST_RESC_NAME_FLAG     0x2
-// BACKUP_RESC_NAME has been deprecated.
-#define BACKUP_RESC_NAME_FLAG   0x4
+// BACKUP_RESC_NAME has been removed in iRODS 5.0.
+// The keyword is left so we do not reuse old values.
+// #define BACKUP_RESC_NAME_FLAG   0x4 // Removed in iRODS 5.0
```

Comment:
These lines mention removal twice. Let's pick one and remove the other - either L97 or the trailing comment on L99. The choice is yours on which one you prefer.

Reply:
Remove lines 97 and 98? Line 98 implied by 99?

Reply:
Done.